frontend/src/app/workflows/workflows.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="btn-toolbar mb-3 mx-2" role="toolbar" aria-label="Toolbar">
+  <div ngbDropdown>
+    <button class="btn btn-primary m-1" id="exampleSelection"
+            ngbDropdownToggle [disabled]="isLoading()">{{currentExampleDesc}}</button>
+    <div ngbDropdownMenu aria-labelledby="exampleSelection">
+      <ng-container *ngFor="let example of examplesList; index as i">
+        <button ngbDropdownItem (click)="changeExampleContent(i)">{{example}}</button>
+      </ng-container>
+    </div>
+  </div>
+
+  <div *ngIf="!isLoading()">
+    <button class="btn btn-primary m-1"
+            (click)="generate()">Generate
+    </button>
+  </div>
+
+  <div *ngIf="isLoading()">
+    <button class="btn btn-primary m-1" type="button" disabled>
+      <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+      Generating...
+    </button>
+  </div>
+
+  <div>
+    <button class="btn btn-success m-1"
+            (click)="download.open()">
+      Download
+    </button>
+  </div>
+</div>
+
+<!-- Main Content Area -->
+<div class="vh-100">
+  <as-split #split
+            direction="horizontal"
+            (dragStart)="dragStartHandler()"
+            (dragEnd)="dragEndHandler()"
+            (gutterClick)="splitGutterClick($event)">
+    <as-split-area [size]="30">
+      <app-autotheme-codemirror [(content)]="content" [options]="codemirrorOptions"></app-autotheme-codemirror>
+    </as-split-area>
+
+    <as-split-area [size]="70">
+      <as-split
+        direction="vertical"
+        (dragStart)="dragStartHandler()"
+        (dragEnd)="dragEndHandler()"
+        (gutterClick)="splitGutterClick($event)">
+        <as-split-area [size]="35">
+          <div class="h-100" style="background-color: white">
```

Comment:
No inline style necessary, Bootstrap has bg-white.

Reply:
Is removed for both iFrames